mitmproxy/web/app.py

```diff
@@ -364,7 +367,8 @@ def get(self):
                 anticomp=self.master.options.anticomp,
                 stickyauth=self.master.options.stickyauth,
                 stickycookie=self.master.options.stickycookie,
-                stream= self.master.options.stream_large_bodies
+                stream= self.master.options.stream_large_bodies,
+                contentViews= map(lambda v : v.name, contentviews.views)
```

Comment:
`[v.name for v in contentviews.views]`
(http://www.artima.com/weblogs/viewpost.jsp?thread=98196)